README.md

```diff
@@ -25,10 +26,41 @@ $ export REGISTRY=<SOME_REGISTRY> \
 ## Using the Marketplace Operator
 
 ### Description
+The operator manages two CRDs: [OperatorSource](./deploy/crds/operatorsource.crd.yaml) and [CatalogSourceConfig](./deploy/crds/catalogsourceconfig.crd.yaml).
 
-The marketplace operator manages two CRDs: [OperatorSource](./deploy/crd/operatorsource.crd.yaml) and [CatalogSourceConfig](./deploy/crd/catalogsourceconfig.crd.yaml). When an OperatorSource CR is created in the same namespace as where the marketplace operator is running (we recommend the namespace be called "openshift-marketplace"), the operator will download artifacts stored in the registry specified in this OperatorSource CR (for now, please see documentation about using [quay](https://quay.io)'s appregistry API). For an example of this OperatorSource CR please see the [examples](./deploy/examples/) folder.
+`OperatorSource` is used to define the external datastore we are using to store operator artifacts. At the moment we only support Quay's app-registry as our external datastore. Here is an example `OperatorSource`:
+```
+apiVersion: marketplace.redhat.com/v1alpha1
+kind: OperatorSource
+metadata:
+  name: my-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: my-quay-appregistry-namesapce
+  displayName: "My Operators"
+  publisher: "Your name or company"
+```
+On adding an `OperatorSource` to an OKD cluster, they will be visible in the [OperatorHub UI](https://github.com/openshift/console/tree/master/frontend/public/components/marketplace) in the OKD console. There is no equivalent UI in the Kubernetes console. 
+
+`CatalogSourceConfig` is used to install an operator present in the `OperatorSource` to your cluster. Behind the scenes, it will configure an OLM `CatalogSource` so that the operator can then be managed by OLM. Here is an example `CatalogSourceConfig`:
+```
+apiVersion: marketplace.redhat.com/v1alpha1
+kind: CatalogSourceConfig
+metadata:
+  name: my-operators
+  namespace: openshift-marketplace
+spec:
+  targetNamespace: namespace-that-olm-watches
+  packages: my-operator
+  csDisplayName: "My Operators"
+  csPublisher: "Your name or company"
+```
```

Comment:
Same comment as above about the operatorsource cr

Reply:
Same response as above.